CHANGELOG.md

```diff
@@ -2,10 +2,15 @@
 
 * Updated Realm Core to 0.100.2.
 
+### Breaking changes
+
+* Removed `Realm.executeTransaction(transaction, callback)`. Use `Realm.executeTransactionAsync()` instead.
```

Comment:
Yes, we shouldn't remove methods in patch releases, even if they have been deprecated.